com.lunarclient.LunarClient.yml

```diff
@@ -0,0 +1,69 @@
+app-id: com.lunarclient.LunarClient
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: lunarclient
+separate-locales: false
+rename-icon: lunarclient
+finish-args:
+  - --persist=.minecraft
+  - --persist=.lunarclient
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
```

Comment:
Does it actually support Wayland?

Reply:
It does not at least without a custom glfw which this launcher does not allow to be used

Reply:
Launcher should support Wayland because it's using Chromium. Minecraft itself doesn't support it, but it's possible to swap  glfw x11 for Wayland one. It should be possible for upstream to include patched glfw in Launcher and detect if Wayland is available. https://github.com/Admicos/minecraft-wayland